android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -80,6 +80,7 @@ public void onBindViewHolder(@NonNull WalletCoinAdapter.ViewHolder holder, int p
         // we modifying checkbox. This may cause unwanted modifying of the model
         holder.resetObservers();
 
+        holder.iconImg.setImageResource(android.R.color.transparent);
```

Comment:
do we need to change it for every list that uses that adapter? Mayber we need it only for `SELECT_ACCOUNTS_LIST`?